heidrun/pa_dc.rb

```diff
@@ -0,0 +1,108 @@
+
+# Convert ValueArray to an array to manipulate and then reinstantiate a new ValueArray
+#   If more than one dc:contributor property exists then it drops the last one 
+#   If one or zero exist then do nothing
+# 
+# Feels kludgey as heck, should rewrite once ValueArray supports array sliceing 
+# See ticket https://issues.dp.la/issues/8287
+contributor_top = lambda do |r|
+  # testing 
+  # r['dc:contributor'].first_value( r['dc:contributor'].count-1 )
+  # end test
+
+  if r['dc:contributor'].count > 1 
```

Comment:
This approach looks okay, as you say, until we close 8287.

You might avoid running the `#['dc:contributor']` and `#count` logic repeatedly with something like:

```ruby
contrib = r['dc:contributor']
count   = contrib.count

(count > 1) ? contrib.first_value(count - 1) : contrib 
# ^ or do this as `if; else; end`, if preferred
```

and similarly for the example below.